schema/Claim.schema.yaml

```diff
@@ -0,0 +1,41 @@
+title: Claim
+'@id': stencila:Claim
+extends: Entity
```

Comment:
For consistency with schema.org, and other benefits (see below), this should extend `CreativeWork`.

Reply:
Makes sense!